osquery/CMakeLists.txt

```diff
@@ -98,6 +98,9 @@ ADD_OSQUERY_LINK_CORE("glog")
 ADD_OSQUERY_LINK_CORE("boost_system")
 ADD_OSQUERY_LINK_CORE("boost_filesystem")
 ADD_OSQUERY_LINK_CORE("boost_regex")
+ADD_OSQUERY_LINK_CORE("cryptsetup")
```

Comment:
I've just tested this. Building the core (make sdk) does indeed not require these libraries. A normal build (make) does require them.